src/mysql/create.sql

```diff
@@ -0,0 +1,121 @@
+-- This file will create the database and its tables
+
+CREATE DATABASE Mintern;
+USE Mintern;
+
+CREATE TABLE Major (
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(255),
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE User (
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(255),
+  email VARCHAR(255),
+  major_id INT NOT NULL,
+  is_mentor BOOLEAN,
+  PRIMARY KEY (id),
+  FOREIGN KEY (major_id) 
+  REFERENCES Major (id)
+);
+
+CREATE TABLE Notification (
+  id INT NOT NULL AUTO_INCREMENT,
+  user_id INT NOT NULL,
+  message TEXT,
+  url VARCHAR(255),
+  date_time DATETIME,
+  PRIMARY KEY (id),
+  FOREIGN KEY (user_id) 
+  REFERENCES User (id)
+);
+
+CREATE TABLE Tag (
```

Comment:
Can you either rename this or add a comment explaining what a Tag is? It's kind of confusing with QuestionTag below.

Reply:
done